SegmentedControlTab.js

```diff
@@ -133,15 +146,19 @@ SegmentedControlTab.propTypes = {
     borderRadius: PropTypes.number,
     textNumberOfLines: PropTypes.number,
     allowFontScaling: PropTypes.bool,
+    accessible: PropTypes.bool,
+    accessibilityLabel: PropTypes.array,
```

Comment:
it should be `accessibilityLabels` as it will be an array of values. 

The other approach is we could think of having it as a string instead of `array`. And in `line 4o` in tab option we can have it has accessibilityLabel={`${accessibilityLabel} ${text}`}.

Reply:
agree with your first line statement of having it as `accessibilityLabels`. 

Not convinced with having it as a `string` and I didn't understand why you need to use this `accessibilityLabel={${accessibilityLabel} ${text}}` ?

Reply:
Having it as an array is also fine but to have consistency the  `accessibilityLabels` will always be equal to the `values` props. if the consumer misses a value in `accessibilityLabels` ex`accessibilityLabels = { ["tab one", "tab two"] }` values={["one", "two", "three"]} in this case the 1st 2 tabs will have accessibilityLabels and the third one will call out text. that will result in a different behavior to the user.

Option 2 **string**.

In this case, we can give a common value. I.e what extra value you will need other than the text.
ex `accessibilityLabel= "tab"`  `values={["one", "two"]}` 
in this case, the behavior remains consistent output will be  `tab one` & `tab two`.